### PR TITLE
fix(@wallet): left bar selection

### LINKED ISSUE #10999: Wallet -> All accounts: All accounts button can't be selected right after adding  watch only account
### Description

When u add w/o account right after u logged in with new user , then your new w/o account is selected (marked as grey) along with Status account. This leads to inability to click **All accounts** button

Steps:

1. create user
2. open wallet
3. add w/o account `0xea123F7beFF45E3C9fdF54B324c29DBdA14a639A`
4. do not click anywhere and notice the accounts selection

**As result**, both accounts are selected.

Screenshot 2023-06-08 at 17 28 48

**Expected**: only currently shown account is selected on UI

https://github.com/status-im/status-desktop/assets/82375995/1cb5f5b2-dfdf-41b7-8fbd-1aa0635652c2

### PR DESCRIPTION
**Pull request by alaibe**
fixes #10999

### CODE REVIEW

File: src/app/modules/main/wallet_section/module.nim [hunk 1]
```diff
@@ -137,6 +137,7 @@ method notifyFilterChanged(self: Module) =
   self.transactionsModule.filterChanged(self.filter.addresses, self.filter.chainIds)
   self.accountsModule.filterChanged(self.filter.addresses, self.filter.chainIds, self.filter.excludeWatchOnly)
   self.sendModule.filterChanged(self.filter.addresses, self.filter.chainIds)
+  self.view.filterChanged(self.filter.addresses[0], self.filter.excludeWatchOnly, self.filter.allAddresses)
```
Comment (Cuteivist) [inline src/app/modules/main/wallet_section/module.nim:140]: I know it's temporary but it's worth checking if `addresses` is not empty

File: ui/app/AppLayouts/Wallet/views/LeftTabView.qml [hunk 2]
```diff
@@ -125,6 +125,12 @@ Rectangle {
         function onDestroyAddAccountPopup() {
             addAccount.active = false
         }
+        function onFilterChanged(address, excludeWatchOnly, allAddresses) {
+            if (!allAddresses) {
```
Comment (Cuteivist) [inline ui/app/AppLayouts/Wallet/views/LeftTabView.qml:129]:
```suggestion
            root.currentAddress = allAddresses ? "" : address
```

**Review by Cuteivist — COMMENTED** [review]
Great job. Just small minor tweaks

File: src/app/modules/main/wallet_section/module.nim (hunk 1 above)
Comment (alaibe) [reply src/app/modules/main/wallet_section/module.nim:140]: Not temporary on this one :)

**Review by stefandunca — APPROVED** [review]

**Review by Khushboo-dev-cpp — APPROVED** [review]

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**